kwargs.jl

```diff
@@ -0,0 +1,37 @@
+module Kwargs
+
+include(joinpath(@__DIR__, "autodetect-dependabot.jl"))
+
+function kwargs(; coverage::Bool,
+                  force_latest_compatible_version::Union{Bool, Symbol})
+    is_dependabot_job = AutodetectDependabot.is_dependabot_job()
+    kwargs_dict = Dict{Symbol, Any}()
+    kwargs_dict[:coverage] = coverage
+
+    if !Sys.isapple() && VERSION >= v"1.7.0-" # TODO: delete this line
+    # if VERSION >= v"1.7.0-" # excludes 1.6, includes 1.7-DEV, includes 1.7 # TODO: uncomment this line
+        if force_latest_compatible_version isa Bool
+            kwargs_dict[:force_latest_compatible_version] = force_latest_compatible_version
+        elseif force_latest_compatible_version == :auto
+            if is_dependabot_job
+                @info("This is a Dependabot/CompatHelper job, so `force_latest_compatible_version` has been set to `true`")
+                kwargs_dict[:force_latest_compatible_version] = true
+            else
+                kwargs_dict[:force_latest_compatible_version] = false
+            end
```

Comment:
```suggestion
            is_dependabot_job && @info("This is a Dependabot/CompatHelper job, so `force_latest_compatible_version` has been set to `true`")
            kwargs_dict[:force_latest_compatible_version] = is_dependabot_job
```

(Untested and written in the GH suggestion prompt)

Remove another nesting level here will also help make understanding what's going on easier, I think.